eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/test/fasta/FastaSequenceReaderTest.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package uk.ac.ebi.eva.accession.pipeline.io;
+package uk.ac.ebi.eva.accession.core.test.fasta;
```

Comment:
The usual practice is to keep each test class in the same package than the class that is being tested. In this case, this test should be in  uk.ac.ebi.eva.accession.core.io. The existence of a "test" package in this module is confusing and can make you think you should leave your tests there, but I think that package is used for configuration classes.